src/domain/attendanceList/__tests__/AttendanceListPage.test.tsx

```diff
@@ -239,3 +240,42 @@ test('should route to signups page when clicking view participants button', asyn
     expect(mockRouter.asPath).toBe('/registration/registration:1/signup')
   );
 });
+
+test('should export signups as an excel after clicking export as excel button', async () => {
```

Comment:
This test's code seems to be duplicated

Reply:
Refactored test to get rid of the duplicated code.